config/locales/views/automated_tests/en.yml

```diff
@@ -19,6 +20,7 @@ en:
       problems: "error"
     title: "Automated Testing"
     display_output_title: "Display test output to students?"
+    no_test_grouping_for_solution_files: "Unexpected error happened while loading solution files."
```

Comment:
This doesn't really describe the error. Why not display the actual validation error messages instead?